build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
I guess the fundamental flaw here is that we run tests for dnx451 differently than dnxcore50. I actually thought about adding the `dnvmUse` parameter but what the default value would be for dnx451? Wouldn't using the `default` runtime is a change in the behavior that affects all the repos? 
Interestingly dnx repo is adding the path to the runtime before running the test which would work if we ran CoreClr tests the same way we run Desktop Clr tests.